melos.yaml

```
@@ -0,0 +1,46 @@
+  format:
+    description: Format all packages
+    run: melos exec -- "dart format ."
+    
+  # Run tests for all packages
+  test:
+    description: Run tests for all packages
+    run: melos exec --fail-fast --concurrency=1 -- "flutter test"    
+
+environment:
+  sdk: ">=3.6.0 <4.0.0"
+  flutter: ">=3.0.0"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)